### PR TITLE
Dev mode for comsat

### PR DESCRIPTION
**Pull request by dbusel**

### CODE REVIEW

File: src/server/services/comsatService.js [hunk 1]
```diff
@@ -69,7 +70,7 @@ const openPortsOnComsat = function (params, callback) {
     
     let options = {
         host: params.satellite.domain,
-        port: 8080,
+        port: (params.satellite.devMode == 1) ? constants.HTTP_PORT : constants.HTTPS_PORT,
```
Comment (alexandershpak) [inline src/server/services/comsatService.js]: extract variable

File: src/server/services/comsatService.js [hunk 2]
```diff
@@ -154,7 +130,7 @@ const closePortsOnComsat = function (params, callback) {
 
             let options = {
                 host: obj.domain,
-                port: 8080,
+                port: (obj.devMode == 1) ? constants.HTTP_PORT : constants.HTTPS_PORT,
```
Comment (alexandershpak) [inline src/server/services/comsatService.js]: extract variable

File: src/server/services/comsatService.js [hunk 3]
```diff
@@ -238,7 +191,7 @@ const closePortOnComsat = function (params, callback) {
     
     let options = {
         host: params.satellite.domain,
-        port: 8080,
+        port: (params.satellite.devMode == 1) ? constants.HTTP_PORT : constants.HTTPS_PORT,
```
Comment (alexandershpak) [inline src/server/services/comsatService.js]: extract variable

File: src/server/services/comsatService.js [hunk 4]
```diff
@@ -354,11 +285,11 @@ const verifyComsatConnections = function (params, callback) {
         process.stdout.clearLine();
         process.stdout.cursorTo(0);
         percentage_done = Math.round((count / params.satellite.length) * 100);
-
         process.stdout.write('Percentage completed ' + percentage_done + '%');
+
         let options = {
             host: satellite.domain,
-            port: 8080,
+            port: (satellite.devMode == 1) ? constants.HTTP_PORT : constants.HTTPS_PORT,
```
Comment (alexandershpak) [inline src/server/services/comsatService.js]: extract variable

**Comment by epankou** [comment]
Didn't find changes related to CLI comsat help.
Also edit README accordingly

**Pull request merged**